Brain Image Library.md

```diff
@@ -0,0 +1,59 @@
+# Brain Image Library
+## Description
+The Brain Image Library (BIL) will support the goals of the BRAIN Initiative
+Cell Census Network (BICCN) by providing a portal to the entire set of
+images submitted by the participating BICCN labs.  High-performance
+computational resources with high-bandwidth access to this library are
+available both through BIL and the XSEDE program, and will allow analysis
+of these datasets at scale.
+
+# Roles
+[Primary Investigator] Alex Ropelewski (ropelews@psc.edu)
+[System Hardware/Software] Greg Hood (ghood@psc.edu)
+[Website/Submission Portal] Luke Tuite (ltuite96@gmail.com)
+[other names to be added shortly]
+
+## Objectives
+* Construct and operate a large-scale storage platform suitable for housing
+  many thousands of image datasets totaling many petabytes in size.
+
+* Host a metadata portal to allow searching through the metadata of
+  these datasets.
+
+* Host an image portal to allow rapid access to images from these datasets.
+
+## In-scope
```

Comment:
Does this include pipelines for conversion to standardized formats? This is something BIL has expressed interest in doing

Reply:
Yes, this is address by the last item under In-scope.